platforms/f1/build-bitstream.sh

```diff
@@ -1,9 +1,44 @@
 #!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Unless you have good reason for it to be `/bin/bash`